alerting/config.go

```diff
@@ -85,7 +89,8 @@ func (config *Config) GetAlertingProviderByAlertType(alertType alert.Type) provi
 	entityType := reflect.TypeOf(config).Elem()
 	for i := 0; i < entityType.NumField(); i++ {
 		field := entityType.Field(i)
-		if strings.ToLower(field.Name) == string(alertType) {
+		tag := strings.Split(field.Tag.Get("yaml"), ",")[0]
```

Comment:
@TwiN I had to update this because the name of the struct property was checked against instead of the yaml key itself. And `-` is not allowed there.

Hope this is correct, if not let me know how to change it, or you could update it yourself, if you prefer!

Reply:
Or we could revert to naming it `awsses`.

Reply:
Oh dang, that's an excellent catch! Did any of the tests catch it?

Reply:
They did not, no.